notebooks/create_source_data.py

```
@@ -0,0 +1,34 @@
+import pandas as pd
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import current_timestamp, to_utc_timestamp
+
+from src import config, logger
```

Ruff (0.8.0)

`src.logger` imported but unused

Remove unused import: `src.logger`

(F401)